ci/test_narwhals.sh

```diff
@@ -17,7 +17,7 @@ set +e
 rapids-logger "pytest narwhals"
 git clone https://github.com/narwhals-dev/narwhals --depth=1 -b stable
 pushd narwhals || exit 1
-rapids-pip-retry install -U -e ".[dev]"
+rapids-pip-retry install -U -e ".[dev]" pytest-env hypothesis
```

Comment:
```suggestion
rapids-pip-retry install -U -e . pytest-env hypothesis
```
Thanks again for spotting this!
Small note: `dev` shouldn't have any effect here. it's not an optional dep anymore

Reply:
thanks!